gtsfm/frontend/correspondence_generator/correspondence_generator_base.py

```diff
@@ -16,14 +16,14 @@ class CorrespondenceGeneratorBase:
     def create_computation_graph(
         self,
         delayed_images: List[Delayed],
-        image_shapes: List[Tuple[int, int]],
+        delayed_image_shapes: List[Delayed],
```

Comment:
Why make them delayed? I think we reverted to using non-delayed types during Hilti challenge.

Reply:
they are already delayed, I am only fixing the type hints here :)

Reply:
the reason we made them delayed in a different PR was so that they could be accessed on a cluster. I dont think they need to be delayed though.. that would be on @stepanyanhayk  to check.

Reply:
Confirming that the image shapes to the CorrespondenceGenerator are currently delayed, according to this line: https://github.com/borglab/gtsfm/blob/master/gtsfm/runner/gtsfm_runner_base.py#L232